bodhi/server/views/generic.py

```diff
@@ -67,6 +67,76 @@ def get_latest_updates(request, critpath, security):
     query = query.order_by(models.Update.date_submitted.desc())
     return query.limit(5).all()
 
+def _get_status_counts(basequery, status):
+    """
+    Return a dictionary with the counts of objects found in the basequery,
+    specified by total count, newpackage count, bugfix count, enhancement count,
+    and security count. The dictionary keys will be named with the
+    template {status}_{type}_total. For example, if status is
+    models.UpdateStatus.stable, a dictionary with the following keys would be
+    returned:
+        stable_updates_total
+        stable_newpackage_total
+        stable_bugfix_total
+        stable_enhancement_total
+        stable_security_total
+
+    Args:
+        basequery
+        status (bodhi.server.models.UpdateStatus): The update status we want to filter by in basequery
```

Comment:
The automated PEP-8 tool also enforces that lines can't be over 100 characters, and this line is 102 characters:

```
/home/vagrant/bodhi/bodhi/server/views/generic.py:86:101: E501 line too long (102 > 100 characters)
```